code/game/objects/items/weapons/twohanded.dm

```diff
@@ -435,11 +436,18 @@
 	if(wielded)
 		. = ..()
 
-
 /obj/item/weapon/twohanded/required/chainsaw/doomslayer
 	name = "OOOH BABY"
 	desc = "<span class='warning'>VRRRRRRR!!!</span>"
 	armour_penetration = 100
+	force_on = 30
+
+/obj/item/weapon/twohanded/required/chainsaw/doomslayer/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance, damage, attack_type)
+	if(attack_type == PROJECTILE_ATTACK)
+		owner.visible_message("<span class='danger'>Ranged attacks just make [owner] angrier!</span>")
```

Comment:
Add a message to the owner himself in BIG RED TEXT saying how much ANGRIER they are! (The spam serves to actually anger the player)